R/minimize.R

```diff
@@ -35,6 +35,116 @@ minimize <- function(
 
     args <- c(as.list(environment()), list(...))
 
+
+    if(is(initial_design, "OneStageDesign")) {
+        res <- do.call(.minimize_os, args = args)
+
+    } else{
+        if(is.null(lower_boundary_design) || is.null(upper_boundary_design)) {
```

Comment:
as I said, if you want to get automatic boundaries, define separate functions in util.R and use the structure outlined in #145 - or are there any technical issues that I overlooked? Keeps minimize much cleaner!